src/game.rs

```diff
@@ -42,19 +42,10 @@ impl Deserialize for PlayerId {
 ///
 /// Meant to be managed as application state by Rocket. Only one should ever be created, and Rocket
 /// ensures that only one can ever be registered as managed state.
-#[derive(Debug)]
+#[derive(Debug, Default)]
 pub struct PlayerIdGenerator(AtomicUsize);
 
 impl PlayerIdGenerator {
-    /// Creates a new `PlayerIdGenerator`.
-    ///
-    /// Only one `PlayerIdGenerator` should be created in the lifetime of the application. A single
-    /// generator will never create duplicate IDs, but if there are multiple generators will
-    /// produce the same IDs.
-    pub fn new() -> PlayerIdGenerator {
-        PlayerIdGenerator(ATOMIC_USIZE_INIT)
```

Comment:
Functionally, the `new` constructor (that I removed) does the same thing as the derived `Default::default()` constructor (that I added) is doing. They both initialize the `AtomicUsize` as 0. Generally it's best practice to have both a `Default` impl and a `new` constructor, but in this case we can write less code by automatically deriving the `Default` impl.